.github/workflows/main.yml

```diff
@@ -66,14 +66,13 @@ jobs:
           COMPOSER_MEMORY_LIMIT=-1 composer require civicrm/civicrm-asset-plugin:'~1.1' civicrm/civicrm-{core,packages}:${{ matrix.civicrm }} --no-suggest --prefer-dist
       # For some reason drupal/webform:5.x installs even if it is drupal:^9.0
       - name: Ensure Webform ^6.0
-        if: matrix.drupal == '^9.0'
```

Comment:
So we're dropping testing for Webform 5.x?

Reply:
As it turns out testing on GitHub Actions was already happening on 6.x.x only (even on the D8.9 -> composer was already pulling in 6.x.x).

Reply:
And since both are `core_version_requirement: ^8.8 || ^9` - we can drop support for 5.x